src/App.tsx

```diff
@@ -41,9 +42,9 @@ const App: React.FC = () => {
 
   useEffect(() => {
```

Comment:
isn't it redundant with `useEagerConnect` ?

Reply:
Correct